src/types/value/linkToMedia.ts

```diff
@@ -25,4 +25,5 @@ export interface FilledLinkToMediaField {
 	height?: string | null
 	width?: string | null
 	text?: string
+	variant?: string
```

Comment:
⚠️ #issue: From [this Slack message](https://prismic-team.slack.com/archives/C02L3FN3AJK/p1733937871356369?thread_ts=1733933751.211369&cid=C02L3FN3AJK), it seems we are only supporting `variant` on link fields, not content relationship and link to media fields.